content/content-bundles/resources/types.properties

```diff
@@ -14,6 +14,7 @@ access.groups8		 = Resources in the ''{0}'' folder can only be viewed by the gro
 access.groups9		 = Resources in the ''{0}'' folder can only be viewed by the groups <strong>''{1}'', ''{2}'', ''{3}'', ''{4}'', ''{5}'', ''{6}'', ''{7}'', ''{8}'' and ''{9}''</strong>. You can further restrict this file by group if you wish.
 access.groupsX		 = Resources in the ''{0}'' folder can only be viewed by the groups <strong>''{1}'', ''{2}'', ''{3}'', ''{4}'', ''{5}'', ''{6}'', ''{7}'', ''{8}'', ''{9}'' and others</strong>. You can further restrict this file by group if you wish.
 access.inherit		 = Leave access settings as for the folder.
+access.secured = Allow secured access to this file. This file can be accessed by students in special contexts like an assessment attempt.
```

Comment:
Secure access to this resource only restricting access in specific conditions.

Reply:
Done